### PR TITLE
fanju

### PR DESCRIPTION
**Pull request by danilkorotkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ Works with Digoo DG-R8H 433MHz Wireless Digital Hygrometer Thermometer Weather S
 
 This plugin should work with a broad number of OOK based devices such as power plugs and remotes
 
+### fanju 
+
```
Comment (dgomes) [inline README.md:42]: Please add a sentence explaining what it is, in the reason to link these 2 websites

File: binary.h [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/*
+  Copyright (C) 2013 - 2014 CurlyMo
+
+  This file is part of pilight.
+
+  pilight is free software: you can redistribute it and/or modify it under the
```
Comment (dgomes) [inline binary.h:6]: Not sure we can add a file protected by GPL to an LGPL project :(

File: fanju.cpp [hunk 3]
```diff
@@ -0,0 +1,168 @@
+/*
+  Written by Diogo Gomes, diogogomes@gmail.com
+
+  Copyright (c) 2014 Diogo Gomes.  All right reserved.
```
Comment (dgomes) [inline fanju.cpp:4]: Please put your name :) not mine

File: fanju.cpp [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        offset = i + 1;
+        break;
+      }
+    } else {
+      sync = 0;
+      return;
+    }
+  }
+ 
+  //Serial.print("offset ");Serial.println(offset);
```
Comment (dgomes) [inline fanju.cpp:97]: lets clean this code out

File: binary.h (hunk 2 above)
Comment (danilkorotkov) [reply binary.h:6]: i think we can. We just use it in the open project without any modifications.

**Review by dgomes — APPROVED** [review]
Thanks!

**Pull request merged**